assets/js/ConfiguredHeadwaysForm.tsx

```diff
@@ -113,128 +138,98 @@ function ConfiguredHeadwaysForm({
         {},
       );
       setConfiguredHeadways(newConfig);
+      reset({}, { keepValues: true });
     },
     [setConfiguredHeadways],
   );
-
   return (
     <div className="mb-5">
-      <Formik<{
-        branches: {
-          id: string;
-          [timePeriodId: string]:
-            | {
-                range_high: number;
-                range_low: number;
-              }
-            | string;
-        }[];
-      }>
-        validationSchema={formSchema}
-        enableReinitialize
-        initialValues={initialFormValues}
-        onSubmit={handleSubmit}
-        validateOnBlur
-        validateOnChange={false}
+      {errors.branches ? (
```

Comment:
I just tested this locally, and noticed some odd behavior around when the error message displays. Sometimes it doesn't show up after making an invalid edit to one of the two fields, and sometimes it persists even after the form transitions to valid. I didn't dig in too much, but I suspect it might be related to how we're using `errors.branches` here. Maybe this should be `!isValid` instead?

Reply:
Ah yep I think you're right. Changed it to use `!isValid` and seems to work much more consistently. I think I opted to not use that flag at first because when running locally, due to the absence of values, the flag will show up immediately but that actually would be the behavior we want since all the values are required by the schema. All that said, this change would actually be somewhat of a bugfix considering that the current version of Signs UI doesn't show the error banner when running local.

Reply:
I'm confused about that last bit. I've always seen the error banner when running locally.

In any case, I retested and it looks good now.

Reply:
Huh that's weird. For example, here's me running current master and then opening up the headways form:
https://user-images.githubusercontent.com/16074540/225113368-89f00eea-c84e-4660-aa24-3d2e815b8fc2.mov

Reply:
Ah, I misunderstood, I thought you meant you never saw the banner. That vid matches what I saw.